deltacat/utils/resources.py

```diff
@@ -169,6 +170,12 @@ def get_current_process_peak_memory_usage_in_bytes():
         return psutil.Process().memory_info().peak_wset
 
 
+def get_current_process_current_memory_usage_in_bytes():
+    process = psutil.Process(os.getpid())
+    mem_info = process.memory_info()
+    return mem_info
+
+
```

Comment:
Since this isn't currently used can we remove it? Or if we're going to use it for a profiler in a follow-up PR we can keep it for now.